### PR TITLE
Add npm steps to A.js setup [DOC-771]

### PR DESCRIPTION
**Pull request by stayseesong**

### CODE REVIEW

File: src/connections/sources/catalog/libraries/website/javascript/quickstart.md [hunk 1]
```diff
@@ -4,70 +4,109 @@ hidden: true
...
+When you create a source in the Segment web app, it tells the Segment servers that you'll be sending data from a specific source type. When you create (or change) a source in the Segment app, Segment generates a new write key for that source. You use the write key in your website code to tell Segment servers where the data is coming from, so Segment can route it to your Destinations and other tools.
+
+
+## Step 2: Install Segment to your site
+
+You can choose to install Segment to your site in 1 of 2 ways:
+
+[1. Install Segment as a NPM package](#step-2a-install-segment-as-a-npm-package)
+
+[2. Add the Segment snippet to your app](#step-2b-add-the-segment-snippet)
```
Comment (cmastr) [inline src/connections/sources/catalog/libraries/website/javascript/quickstart.md:30]: Maybe these should be a and b to match the headings below?

File: src/connections/sources/catalog/libraries/website/javascript/quickstart.md [hunk 2]
```diff
@@ -109,16 +148,12 @@ analytics.track('Article Bookmarked', {
...
 
-To get started, we recommend that you track just a few important events. You can always add more later!
-
-Once you add a few `track` calls, **you're done with this tutorial!** You successfully installed Analytics.js tracking. Now you're ready to turn on any Destination you like from our interface, margarita in hand.
-
-
----
+To get started, Segment recommends that you track just a few important events. You can always add more later.
 
+Once you add a few Track calls, you successfully installed Analytics.js tracking. Now you're ready to turn on any destination you like from the Segment interface.
```
Comment (cmastr) [inline src/connections/sources/catalog/libraries/website/javascript/quickstart.md:153]:
```suggestion
After you add a few Track calls, you successfully installed Analytics.js tracking. Now you're ready to turn on any destination you like from the Segment interface.
```

File: src/connections/sources/catalog/libraries/website/javascript/quickstart.md [hunk 3]
```diff
@@ -4,70 +4,109 @@ hidden: true
...
 
-When you create a Source in the Segment web app, it tells the Segment servers that you'll be sending data from a specific source type. When you create (or change!) a Source in the Segment app, Segment generates a new Write Key for that source. You use the write key in your website code to tell Segment servers where the data is coming from, so Segment can route it to your Destinations and other tools.
+When you create a source in the Segment web app, it tells the Segment servers that you'll be sending data from a specific source type. When you create (or change) a source in the Segment app, Segment generates a new write key for that source. You use the write key in your website code to tell Segment servers where the data is coming from, so Segment can route it to your Destinations and other tools.
+
+
+## Step 2: Install Segment to your site
+
+You can choose to install Segment to your site in 1 of 2 ways:
+
+[1. Install Segment as a NPM package](#step-2a-install-segment-as-a-npm-package)
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/javascript/quickstart.md:28]:
```suggestion
[a. Install Segment as a NPM package](#step-2a-install-segment-as-a-npm-package)
```

File: src/connections/sources/catalog/libraries/website/javascript/quickstart.md (hunk 1 above)
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/javascript/quickstart.md:30]:
```suggestion
[b. Add the Segment snippet to your app](#step-2b-add-the-segment-snippet)
```

**Review by pwseg — APPROVED** [review]

File: src/connections/sources/catalog/libraries/website/javascript/quickstart.md [hunk 4]
```diff
@@ -4,70 +4,109 @@ hidden: true
...
+
+## Step 2: Install Segment to your site
+
+You can choose to install Segment to your site in 1 of 2 ways:
+
+[a. Install Segment as a NPM package](#step-2a-install-segment-as-a-npm-package)
+
+[b. Add the Segment snippet to your app](#step-2b-add-the-segment-snippet)
+
+### Step 2a: Install Segment as a NPM package
```
Comment (silesky) [inline src/connections/sources/catalog/libraries/website/javascript/quickstart.md:32]: This is kind of a nit but npm isn't actually supposed to be capitalized. 

I generally do: `npm` (with the backticks) to highlight it.

File: src/connections/sources/catalog/libraries/website/javascript/quickstart.md [hunk 5]
```diff
@@ -4,70 +4,109 @@ hidden: true
...
-## Step 2: Add the Segment Snippet
+2. Import and initialize the analytics.
 
-> success "Install Segment as an NPM package"
-> You can add Segment to your project as an [NPM package](https://www.npmjs.com/package/@segment/snippet){:target="_blank"}. For more information, see the instructions on [GitHub](https://github.com/segmentio/analytics-next#-using-as-an-npm-package){:target="_blank"}.
+    ```js
+    import { AnalyticsBrowser } from '@segment/analytics-next'
 
-Paste this snippet into the `<head>` tag of your site to install Segment.
+    export const analytics = AnalyticsBrowser.load({ writeKey: 'YOUR_WRITE_KEY' })
```
Comment (silesky) [inline src/connections/sources/catalog/libraries/website/javascript/quickstart.md:54]: Now that I think about it, we could not document `AnalyticsBrowser.load` here. 
This does everything the other API does, but is more flexible:
```ts
export const analytics = new AnalyticsBrowser()
analytics.load({ writeKey: 'YOUR_WRITE_KEY' })
```

We can probably de-emphasize it on our README docs as well.

File: src/connections/sources/catalog/libraries/website/javascript/quickstart.md [hunk 6]
```diff
@@ -4,70 +4,109 @@ hidden: true
...
+## Step 1: Create a source in the Segment app
 
-Before you begin, you need a Workspace (which is a container that holds all of the Sources and Destinations that are billed together for an organization). If you already created one, great! If not, you can sign up for a free Segment account and create one.
+Before you begin, you need a Workspace (which is a container that holds all of the Sources and Destinations that are billed together for an organization). You can sign up for a free Segment account and create a workspace.
 
-Next create an Analytics.js source from your Workspace:
-
-1. Click **Add Source**.
+To create an Analytics.js source source in the Segment app: 
+1. 1. Click **Add Source**.
```
Comment (silesky) [inline src/connections/sources/catalog/libraries/website/javascript/quickstart.md:16]: This looks a little strange in the markdown preview (did you mean to add 1. 1)?

**Comment by silesky** [comment]
Left a couple comments, but looks great!

**Pull request merged**